core/src/model/Model.java

```diff
@@ -44,6 +50,10 @@ public IPlayerCharacter getPlayer(){
         return player;
     }
 
+    /**
+     * Spawns monsters if there are too few, moves monsters toward the player, checks for player levelup
```

Comment:
This description could possibly be simplified.

Reply:
description too detailed, should be along the lines of "updates the state of the model"